src/ocean/application/components/PidLock.d

```diff
@@ -161,14 +161,14 @@ public struct PidLock
         }
 
         // Clear the any previous contents of the file
-        this.enforcePosix!(ftruncate)(this.lock_fd, 0);
+        (&this).enforcePosix!(ftruncate)((&this).lock_fd, 0);
```

Comment:
Personally I would suggest not adding those in the first place. They are just noise that needs to be removed afterwards, except for a very few of them which can be reviewed case-by-case.

Reply:
That was my first approach (with tweaked d1to2fix) but after trying it in practice I got scared of regressions because cases like `return this` can potentially keep compiling but with a changed meaning. It feels safer to cleanup it in chunks that actually can be reasonably reviewed.